dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheTimeoutServer/server.xml

```diff
@@ -0,0 +1,76 @@
+<server>
+
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>jdbc-4.1</feature>
+        <feature>jndi-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+     <!-- Use this config for sessionCache -->
+     <featureManager>
+        <feature>sessionCache-1.0</feature>
+    </featureManager>   
+    <httpSession maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="1s" reaperPollInterval="30"/>
+    
+    <!-- Use this config for testing sessionDatabase
+    <featureManager>
+        <feature>sessionDatabase-1.0</feature>
+    </featureManager>
+    <httpSessionDatabase id="FATDB" dataSourceRef="DerbyDS"/>
+    <httpSession storageRef="FATDB" maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="1s" reaperPollInterval="30"/>
+     -->
+
+    <library id="HazelcastLib">
+        <file name="${shared.resource.dir}/hazelcast/hazelcast.jar"/>
+    </library>
+
+	<library id="DerbyLib">
+	    <file name="${shared.resource.dir}/derby/derby.jar"/>
+	</library>
+
+    <!-- Used for testing that data source can be stored in a session -->
+	<authData id="DerbyAuth" user="userName" password="userPassword"/>    
+    <dataSource id="DerbyDS" jndiName="jdbc/derby" containerAuthDataRef="DerbyAuth">
+	    <jdbcDriver libraryRef="DerbyLib"/>
+	    <properties.derby.embedded createDatabase="create" databaseName="memory:testdb"/>
+	</dataSource>
+
+	<javaPermission codebase="${shared.resource.dir}/derby/derby.jar" className="java.security.AllPermission"/>
```

Comment:
Looks like the set of `<javaPermission>` configs got duplicated somewhere along the way?  Not harming anything, so we can clean it up later if necessary.